spec/integration/migration_spec.rb

```diff
@@ -569,81 +569,81 @@
     it { is_expected.to include_heredoc(sibling_index_sql) }
     it { is_expected.to include_heredoc(grandchild_index_sql) }
 
-    it 'creates the indices using CONCURRENTLY directive because `algorthim: :concurrently` args are present' do
+    it "creates the indices using CONCURRENTLY directive because `algorthim: :concurrently` args are present" do
       subject
       expect(adapter).to have_received(:execute).with(
-        "CREATE  INDEX CONCURRENTLY \"#{index_prefix}_#{Digest::MD5.hexdigest(grandchild_table_name)[0..6]}\" "\
-        "ON \"#{grandchild_table_name}\" USING hash (\"updated_at\") "\
+        "CREATE  INDEX CONCURRENTLY \"#{index_prefix}_#{Digest::MD5.hexdigest(grandchild_table_name)[0..6]}\" " \
+        "ON \"#{grandchild_table_name}\" USING hash (\"updated_at\") " \
         "WHERE created_at > '#{current_date.to_time.iso8601}'"
       )
       expect(adapter).to have_received(:execute).with(
-        "CREATE  INDEX CONCURRENTLY \"#{index_prefix}_#{Digest::MD5.hexdigest(sibling_table_name)[0..6]}\" "\
-        "ON \"#{sibling_table_name}\" USING hash (\"updated_at\") "\
+        "CREATE  INDEX CONCURRENTLY \"#{index_prefix}_#{Digest::MD5.hexdigest(sibling_table_name)[0..6]}\" " \
+        "ON \"#{sibling_table_name}\" USING hash (\"updated_at\") " \
         "WHERE created_at > '#{current_date.to_time.iso8601}'"
       )
     end
 
-    context 'when unique: true index option is used' do
+    context "when unique: true index option is used" do
       subject(:add_index_on_all_partitions) do
         create_list_partition_of
 
         adapter.add_index_on_all_partitions table_name, "#{table_name}_id", name: index_prefix,
-                                            in_threads: index_threads, algorithm: :concurrently, unique: true
+                                                                            in_threads: index_threads, algorithm: :concurrently, unique: true
       end
 
-      it 'creates a unique index' do
+      it "creates a unique index" do
         subject
         expect(adapter).to have_received(:execute).with(
-          "CREATE UNIQUE INDEX CONCURRENTLY \"#{index_prefix}_#{Digest::MD5.hexdigest(child_table_name)[0..6]}\" "\
+          "CREATE UNIQUE INDEX CONCURRENTLY \"#{index_prefix}_#{Digest::MD5.hexdigest(child_table_name)[0..6]}\" " \
           "ON \"#{child_table_name}\"  (\"#{table_name}_id\")"
         )
       end
     end
 
-    context 'when in_threads: is provided' do
+    context "when in_threads: is provided" do
       let(:index_threads) { ActiveRecord::Base.connection_pool.size - 1 }
 
       before do
         allow(Parallel).to receive(:map).with([child_table_name, sibling_table_name], in_threads: index_threads)
-                             .and_yield(child_table_name).and_yield(sibling_table_name)
+                                        .and_yield(child_table_name).and_yield(sibling_table_name)
```

Comment:
Seems like all of these would look better if we just move the `.with` call to its own line

Reply:
Done